packages/core/src/state-management/state-manager.ts

```diff
@@ -50,10 +50,12 @@ export class StateManager {
    */
   async sync(state$: RunningState, timeoutMillis: number): Promise<void> {
     const tip$ = this.dispatcher.messageBus.queryNetwork(state$.id);
-    const tip = await tip$.pipe(timeoutWith(timeoutMillis, of(undefined))).toPromise();
-    if (tip) {
-      await this._handleTip(state$, tip);
-    }
+    await tip$
```

Comment:
Should this variable be called `tips$` now?

Reply:
It is `Observable<CID>`,  not `Observable<CID[]>` so still `tip$`.

Reply:
I c